Makefile

```diff
@@ -77,6 +77,7 @@ install_hooks:  ## Install required hooks
 	bash ./docker/pre_commit_hook.sh --install
 
 clean: ## Cleans the docker containers and volumes.
+	docker rm $$(docker ps -aq --filter="name=dev-server-run-") || true
```

Comment:
This looks like it'll delete containers based on a name filter, which could be dangerous if developers have non-Oppia Docker containers on their systems. Why do we need this?

Reply:
@U8NWXD When running tests, containers with the name `dev-server-run-*` are created, which aren't deleted by docker compose command.

Reply:
Okay, but there's still a problem because a user could create a container whose name follows the `dev-server-run-*` pattern. We don't want to delete containers that are user-created like this.

Instead, could we add a label (see https://docs.docker.com/reference/cli/docker/compose/run/) when issuing the `docker compose run ...` commands and then use that label to identify containers to delete? The label could be something like `oppia-generated-by-docker-compose-run` that's self-explanatory and unlikely to be chosen by a user for a manually-created container.

Reply:
this still needs to be addressed